build-system/tasks/dist.js

```diff
@@ -130,23 +132,24 @@ async function dist() {
     await formatExtractedMessages();
     await generateFileListing();
   }
-  return exitCtrlcHandler(handlerProcess);
+  if (!argv.watch) {
```

Comment:
Because there was no `--watch` argument for `gulp dist` earlier. (When you're in watch mode, the tasks stays alive looking for changes after the initial build. We want `Ctrl+C` to work then.)